tests/helixprep.proj

```diff
@@ -0,0 +1,152 @@
+<Project ToolsVersion="12.0" DefaultTargets="ArchiveAll" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="ZipFileCreateFromDirectory" AssemblyFile="$(ToolsDir)\net45\Microsoft.DotNet.Build.Tasks.dll"/>
+  
+  <Import Project="dir.props" />
+  <Import Project="..\dir.props" />
```

Comment:
Not part of your PR but it's interesting that the current dir.props doesn't also import the upper-directory dir.props (common pattern)